src/WindowsAzureMessaging/WindowsAzureMessaging/Models/MSNotificationHubMessage.m

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSNotificationHubMessage.h"
+
+@implementation MSNotificationHubMessage
+
+@synthesize title;
+@synthesize message;
+@synthesize badge;
+@synthesize customData;
```

Comment:
Instead of calling it custom data, let's try calling it additionalData so they can get whatever they want from it.

Reply:
done